packages/dds/map/src/test/mocha/map.fuzz.spec.ts

```diff
@@ -130,7 +125,7 @@ function makeGenerator(optionsParam?: Partial<GeneratorOptions>): AsyncGenerator
 	return async (state) => syncGenerator(state);
 }
 
-describe("Map fuzz tests", () => {
+describe.only("Map fuzz tests", () => {
```

Comment:
you can removed this, and line 140. there isn't a need to turn this on, only turn it off once its on by default